pkg/glassnode/glassnode.go

```diff
@@ -0,0 +1,59 @@
+package glassnode
```

Comment:
why separate into glassnode and glassnodeapi packages? what's the difference?

Reply:
I think it is just like the `maxapi`, `ftxapi`,... in `exchange`.